zebrad/src/components/inbound/tests.rs

```diff
@@ -420,11 +436,17 @@ async fn setup(
         sync_status,
         latest_chain_tip,
         chain_tip_change,
+        transaction_sender,
     );
 
     // Enable the mempool
     let _ = mempool_service.enable(&mut recent_syncs).await;
```

Comment:
I think another PR dropped the return value from `enable`:
```suggestion
    mempool_service.enable(&mut recent_syncs).await;
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/2823/commits/cef5b049dbd25ba3c68a8551054adbec5b0e7192